onnxscript/optimizer/evaluator.py

```diff
@@ -55,25 +55,32 @@ class IRContext(Protocol):
     This is a placeholder, subject to simplification when a proper IR is defined.
     """
 
-    def get_input(self, node: onnx.NodeProto, index: int) -> ir.Value | None: ...
+    def get_input(self, node: onnx.NodeProto, index: int) -> ir.Value | None:
+        ...
 
-    def get_output(self, node: onnx.NodeProto, index: int) -> ir.Value | None: ...
+    def get_output(self, node: onnx.NodeProto, index: int) -> ir.Value | None:
+        ...
 
-    def input_const_value(self, node: onnx.NodeProto, index: int) -> ir.ConcreteValue: ...
+    def input_const_value(self, node: onnx.NodeProto, index: int) -> ir.ConcreteValue:
+        ...
 
-    def input_shape(
-        self, node: onnx.NodeProto, index: int
-    ) -> onnx.TensorShapeProto | None: ...
+    def input_shape(self, node: onnx.NodeProto, index: int) -> onnx.TensorShapeProto | None:
+        ...
 
-    def input_type(self, node: onnx.NodeProto, index: int) -> onnx.TypeProto | None: ...
+    def input_type(self, node: onnx.NodeProto, index: int) -> onnx.TypeProto | None:
+        ...
 
-    def input_element_type(self, node: onnx.NodeProto, index: int) -> int | None: ...
+    def input_element_type(self, node: onnx.NodeProto, index: int) -> int | None:
+        ...
 
-    def lookup_version(self, domain: str) -> int: ...
+    def lookup_version(self, domain: str) -> int:
+        ...
 
-    def convert_attributes(self, attributes: Sequence[onnx.AttributeProto]) -> dict: ...
+    def convert_attributes(self, attributes: Sequence[onnx.AttributeProto]) -> dict:
+        ...
 
-    def new_constant(self, name: str, value: Any) -> Sequence[onnx.NodeProto] | None: ...
+    def new_constant(self, name: str, value: Any) -> Sequence[onnx.NodeProto] | None:
```

Comment:
linter in CI wants to revert all this, perhaps sync settings in local with `lintrunner init`?

Reply:
ok, will try that